src/Package.zig

```diff
@@ -490,15 +498,312 @@ const Report = struct {
     }
 };
 
+const FetchLocation = union(enum) {
+    /// The absolute path to a file or directory.
+    /// This may be a file that requires unpacking (such as a .tar.gz),
+    /// or the path to the root directory of a package.
+    file: []const u8,
```

Comment:
Absolute paths are not portable, please avoid absolute paths entirely.

Reply:
Absolute paths have been squashed 👍